src/transactions/create_account_operation.rs

```diff
@@ -0,0 +1,77 @@
+use crate::xdr;
+use crate::database::{db_conn, Account, BASE_RESERVE};
+use crate::stellar_base::strkey;
+use super::utils;
+// TODO: I believe, everything should work without using `prelude`. But it doesn't
+use diesel::prelude::*;
+use diesel::result::Error::NotFound;
+
+pub(crate) fn create_account_operation(source_account: Option<xdr::AccountId>, account_operation: xdr::CreateAccountOp) -> xdr::CreateAccountResultCode {
```

Comment:
@Arkweid why do we have `source_account` as an `Option` here?

Reply:
Well, who is the `Source Account` when I will try to create fresh account for myself?